dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -1486,5 +1490,35 @@ def __init__(self, **kwargs):
         width=os.environ.get('RECORD_BREEZE_WIDTH'),
     )
 
+
+@main.command(
+    name='exec',
+    help='Joins the interactive shell of running airflow container'
+    )
+@option_verbose
+@option_dry_run
+@click.argument('exec_args', nargs=-1, type=click.UNPROCESSED)
+def exec(verbose: bool, dry_run: bool, exec_args: Tuple):
+    container_running = find_airflow_container(verbose=verbose, dry_run=dry_run)
+    cmd_to_run = [
+        "docker",
+        "exec",
+        "-it",
+        container_running,
+        "/opt/airflow/scripts/docker/entrypoint_exec.sh",
+    ]
+
+    if exec_args:
+        cmd_to_run.extend(exec_args)
+    run_command(
+        cmd_to_run,
+        verbose=verbose,
+        dry_run=dry_run,
+        check=False,
+        no_output_dump_on_exception=True,
```

Comment:
```suggestion
        no_output_dump_on_exception=False,
```

I tihnk this is useful if exec fails. I also added a little better handling in case of some run_commands - with "nicer" message but I will handle that when I merge my PR.